### DISCUSSION TITLE
QSL Images backup

### DISCUSSION THREAD

**Discussion by jwussler** [wavelog/wavelog#621] [Q&A] [answered]
Hi all,

I want to take on the task of scamming QSL cards.

Once scanned is there a way to export the data with the images attached to the QSO.

I don't mind setting up a CRON job to export the database to my S3 bucket.

I just don't want to start this laborious task if i can't backup the data

**Comment by HB9HIL** [comment, marked as answer]
Hello @jwussler 

The QSL cards are linked with filename and qso_id directly in the database. To properly backup these you have to backup the whole database. 

We recommend backing up the whole instance containing webroot folder and database. This is the safest way. 

More Information about Backup Strategy you can find here: [https://github.com/wavelog/wavelog/wiki/Backup-Strategies](https://github.com/wavelog/wavelog/wiki/Backup-Strategies)
